app/assets/javascripts/oxalis/model/sagas/annotation_saga.js

```diff
@@ -12,15 +15,16 @@ import {
 import constants from "oxalis/constants";
 import Store from "oxalis/store";
 
-export function* pushAnnotationUpdateAsync(): Generator<*, *, *> {
-  const tracing = yield select(state => state.tracing);
-  yield [
-    call(editAnnotation, tracing.annotationId, tracing.tracingType, {
-      name: tracing.name,
-      isPublic: tracing.isPublic,
-      description: tracing.description,
-    }),
-  ];
+export function* pushAnnotationUpdateAsync(): Saga<void> {
+  const tracing = yield* _select(state => state.tracing);
```

Comment:
Just a reminder to rename the `_select` and `_call` according to how you did it in the save_saga :)